src/Algebra/Structures.agda

```diff
@@ -415,15 +415,22 @@ record IsSemiringWithoutOne (+ * : Op₂ A) (0# : A) : Set (a ⊔ ℓ) where
     zero                  : Zero 0# *
 
   open IsCommutativeMonoid +-isCommutativeMonoid public
-    using (setoid)
+    using (setoid; isEquivalence)
```

Comment:
I'd double down on @Taneb 's suggestions, and remove the `setoid` export entirely. As noted in my longer meta-comment, I think it's a design error for a `Structure` to export a sub-`Bundle`... one to chalk up to my relative inexperience/lack of understanding when raising and then tackling #1917 which caused all this in the first place. hey-ho!